### PR TITLE
add css variable changing for 'clusters-menu-bgc'

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <smalton@mirantis.com>

### CODE REVIEW

File: src/renderer/themes/kontena-light.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "textColorPrimary": "#555555",
     "textColorSecondary": "#51575d",
     "textColorAccent": "#333333",
+    "clusters-menu-bgc": "#afafaf",
```
Comment (ixrock) [inline src/renderer/themes/kontena-light.json]: Should be `camelCase`-d as other names

**Review by ixrock — APPROVED** [review]

File: src/renderer/themes/theme-vars.scss [hunk 2]
```diff
@@ -7,6 +7,7 @@ $primary: var(--primary);
 $textColorPrimary: var(--textColorPrimary);
 $textColorSecondary: var(--textColorSecondary);
 $textColorAccent: var(--textColorAccent);
+$clusters-menu-bgc: var(--clusters-menu-bgc);
```
Comment (ixrock) [inline src/renderer/themes/theme-vars.scss]: Here as well

**Review by ixrock — CHANGES_REQUESTED** [review]

**Review by nevalla — APPROVED** [review]
LGTM

**Comment by nevalla** [comment]
@Nokel81 Is still relevant? If yes, feel free to merge after changing the target branch.

**Comment by aleksfront** [comment]
My guess it can be closed since I do a bit of refactoring in cluster menu and other places. Some colors/vars will be changed.

**Comment by nevalla** [comment]
Okay. I'm closing this now. Let's re-open if required.

**Pull request closed without merging**